WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/history/ScanHistoryViewModel.kt

```diff
@@ -1,14 +1,22 @@
 package org.wordpress.android.ui.jetpack.scan.history
```

Comment:
Minor (🔍): Consider running reformat on this class.

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/13827/commits/9af69b0ded024e77e024bc2781ee8c6d58220006